go/engine/common_test.go

```diff
@@ -374,17 +374,16 @@ func SetupTwoDevicesWithHook(t *testing.T, nm string, hook func(tc *libkb.TestCo
 }
 
 func ResetAccount(tc libkb.TestContext, u *FakeUser) {
-	pps, err := tc.G.LoginState().GetPassphraseStreamWithPassphrase(u.Passphrase)
+	pdpka, err := tc.G.LoginState().ComputeLoginPackage(u.Username)
```

Comment:
Do you think it would be safer for everything below to be in a LoginState() handler to block any other LoginState ops while ResetAccount happening?

I know not in the scope of this ticket.